### PR TITLE
Update air-review-approve-pending-completed-actions.md

### LINKED ISSUE #12621: No mention of what happens when the time is up
[Enter feedback here]

The document tells you how to approve or reject investigations, but investigations have a timeframe for approval. What happens if it is not approved in time? Do they get rejected?
---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 75038ca4-5640-5792-4dd9-0baeedb9ad29
* Version Independent ID: f87ee7c8-1869-295a-6d65-079681545baf
* Content: [Review and manage remediation actions in Microsoft Defender for Office 365](https://learn.microsoft.com/en-us/microsoft-365/security/office-365-security/air-review-approve-pending-completed-actions?view=o365-worldwide)
* Content Source: [microsoft-365/security/office-365-security/air-review-approve-pending-completed-actions.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/office-365-security/air-review-approve-pending-completed-actions.md)
* Service: **microsoft-365-security**
* Sub-service: **mdo**
* GitHub Login: @Dansimp
* Microsoft Alias: **dansimp**

### PR DESCRIPTION
**Pull request by msbemba**
added note about the time frame for pending actions to expire

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/12621

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/office-365-security/air-review-approve-pending-completed-actions.md [hunk 1]
```diff
@@ -46,6 +46,7 @@ There are four different ways to find and take auto investigation actions:
 - [Action center](https://security.microsoft.com/action-center/pending)
 - [Investigation and remediation investigations queue](https://security.microsoft.com/airinvestigation)
 
+
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/air-review-approve-pending-completed-actions.md]:
```suggestion
```

File: microsoft-365/security/office-365-security/air-review-approve-pending-completed-actions.md [hunk 2]
```diff
@@ -64,6 +65,10 @@ There are four different ways to find and take auto investigation actions:
    - Select **Open investigation page** to view more details about the investigation.
    - Select **Approve** to initiate a pending action.
    - Select **Reject** to prevent a pending action from being taken.
+  
+> [!NOTE]
+> Pending actions time out after awaiting approval for one week.
+
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/air-review-approve-pending-completed-actions.md:70]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Dansimp Please merge this PR. Thanks

**Pull request merged**